### ISSUE TITLE
Add original name synonyms for renamed columns?

### ISSUE THREAD

**Issue by rod-glover** [pacificclimate/pycds#36]
For reasons not entirely apparent, many columns in the original database are renamed in the ORM. This leads to less readability when looking at both the ORM and the database itself. It may be worth adding synonyms for these columns. It may not however, repair one of the most annoying issues, which is that SQLAlchemy can't automagically generate natural joins based on same-name columns for column `id`s, because in the table declaration, such a column is uniformly renamed to `id`, but in referencing tables (where it is a FK), it is called by its database name, `<table name>_id`. Tiresome.

**Comment by rod-glover** [comment]
Actually, of course it can remedy that problem. We just make the primary name the `<table name>-id` and the synonym `id`. Backwards compatible and enables automatic natural joins.

**Comment by rod-glover** [comment]
In the interests of just getting on with it, I am removing this issue from milestone https://github.com/pacificclimate/pycds/milestone/1 . This is worth doing, but it's not on the critical path for migration management.

**Comment by jameshiebert** [comment]
That's fine to take it out of the critical path. Personally, I *like* the `id` aliases. It makes for less verbose code. E.g. `Station.id` vs. `Station.station_id`.

**Comment by rod-glover** [comment]
Yeah, I like the `id` aliases too. There are others that I don't like so much. We can eat our cake and have it too, fortunately.

**Comment by rod-glover** [comment]
Upping the importance of this. It is intensely annoying to have to translate between ORM column names and database column names. It comes up fairly often, and it slows work.
